### DISCUSSION TITLE
2 NodeRed instance connecting to 1 Home Assistant

### DISCUSSION THREAD

**Discussion by goldriver2** [zachowj/hass-node-red#249] [Q&A] [answered]
Hi,

I have configured and got to work NodeRed in my with the Addon, I would like to connect a second NoreRed instance I have running on another server to my HA .

I have installed the second instance together with the latest “node-red-contrib-home-assistant-websocket” nodes on my second computer.

When setting the new serveron the non HA machine, NodeRed can see the HA instance when I select the search tool and I selected it and I also created a new Access Token in HA for this new instance but it does not connect to HA and I can’t see the list if entities.

I have read on another thread that selecting the “Accept Unauthorized SSL Certificates” have fixed the problems for some but not in my case.

Any idea aht I am doing wrong ?

**Comment by zachowj** [comment]
What do the NR logs show?

**Reply by goldriver2** [reply]
I have tried to find the log for NodeRed but can't find them, (I have NodeRed install on windows and running via the scheduler as per NodeRed installation guide https://nodered.org/docs/getting-started/windows where it is start by node-red.cmd, maybe you have an idea where I could find them ?

**Reply by goldriver2** [reply]
here is what I have in the log :
3 Feb 16:20:29 - [info] [server:Home Assistant] Connecting to http://192.168.1.147:8123
3 Feb 16:20:41 - [info] Stopping modified flows
3 Feb 16:20:41 - [info] [server:Home Assistant] Closing connection to http://192.168.1.147:8123
3 Feb 16:20:41 - [info] Stopped modified flows
3 Feb 16:20:41 - [info] Updated flows
3 Feb 16:20:41 - [info] Starting modified flows
3 Feb 16:20:41 - [error] [server-state-changed:a1ff7844fc98aab5] ConfigError: Entity ID is required
3 Feb 16:20:41 - [info] Started modified flows
3 Feb 16:20:41 - [info] [server:Home Assistant] Connecting to http://192.168.1.147:8123
3 Feb 16:21:07 - [info] [server:Home Assistant] Connecting to http://192.168.1.147:8123
3 Feb 16:21:33 - [info] [server:Home Assistant] Connecting to http://192.168.1.147:8123
3 Feb 16:21:59 - [info] [server:Home Assistant] Connecting to http://192.168.1.147:8123
3 Feb 16:22:25 - [info] [server:Home Assistant] Connecting to http://192.168.1.147:8123

**Reply by goldriver2** [reply]
Any idea ?

**Reply by zachowj** [reply]
Based on the information the Home Assistant nodes don't have access to that IP and port.

You can try importing the flow below and test if the NR has access. If it has access it should return a `statusCode` of 200.

```
[{"id":"6a1af03beb3ab0da","type":"inject","z":"120358abd7c22d30","name":"","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"","payloadType":"date","x":172,"y":7600,"wires":[["4922901024b5406a"]]},{"id":"3bc4afb4063d7dba","type":"debug","z":"120358abd7c22d30","name":"debug 161","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","statusVal":"","statusType":"auto","x":534,"y":7600,"wires":[]},{"id":"4922901024b5406a","type":"http request","z":"120358abd7c22d30","name":"","method":"GET","ret":"txt","paytoqs":"ignore","url":"http://192.168.1.147:8123","tls":"","persist":false,"proxy":"","insecureHTTPParser":false,"authType":"","senderr":false,"headers":[],"x":346,"y":7600,"wires":[["3bc4afb4063d7dba"]]}]
```

If not then you have some networking issues.

**Reply by goldriver2** [reply, marked as answer]
You got it, I had a firewall rule in place for years that I forgot preventing these 2 specific computers to exchange... I have to find a contour route.
thanks and sorry for thr invonvenience
